backend/db_migrations/0002_initial-model.sql

```diff
@@ -1,7 +1,9 @@
 CREATE TABLE IF NOT EXISTS app.project (
   id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
-  description text NOT NULL,
   project_name text NOT NULL,
+  description text NOT NULL,
+  start_date DATE NOT NULL,
```

Comment:
[nit] The migrations library seems to check all hashes and fail if some of the existing migration files have a different hash from the executed ones.

I think it's fine to let this one pass to get these columns ordered properly, but the table and the old migration row should be dropped locally and in Azure to get rid of errors. Maybe after this one every change to a table should be in its own migration file?